test/Microsoft.ML.StaticPipelineTesting/StaticPipeTests.cs

```diff
@@ -844,20 +843,20 @@ public void TextNormalizeStatic()
 
             Assert.True(schema.TryGetColumnIndex("norm", out int norm));
             var type = schema.GetColumnType(norm);
-            Assert.True(!type.IsVector && type.ItemType.IsText);
```

Comment:
When I read this I think we're probably doing a good thing by removing the conveniences, since I feel like while they *can* be convenient they can also lead to confusion. Here we have someone that tested to make sure something wasn't *vector* and that it was also text, thinking that perhaps `IsText` would return for a vector of text (a totally reasonable expectation), when in fact it is just some shorthand for `is TextType`.